### ISSUE TITLE
[EPIC] Developer Database

### ISSUE THREAD

**Issue by cjd425** [near/devrel#45]
# Proposal for: Ensure we have a searchable database of the active contributors in our ecosystem.

## What will be accomplished by completing this and who will it impact?
- See next steps at the bottom to see what will be accomplished. It will impact everyone interfacing with the community. 

## Which OKR(s) do this relate to?

- [x] 1. Increase the number of **successful** apps running on MainNet
- [x] 2.  Increase the number of **successful** developers deploying and improving their apps on TestNet
- [x] 3. Support the continued maintenance of the materials that will allow devs to self-serve and be **successful**

## What resources do you need to accomplish this?
- (delete this line) describe what resources you think you might or will need to accomplish this.

## What t-shirt size is this?
- [x] x-large (needs to be broken into smaller tasks, most likely an Epic)
- [ ] large (whole sprint & shouldn't be done alone)
- [ ] medium (anything half a week to a week)
- [ ] small (anything from half a day to a full day)
- [ ] x-small (less than half a day)

## What priority is this?
- [ ] P0 (EMERGENCY: drop everything to put out the fire)
- [X] P1 (Critical: docs, bugs, or features that block work or dev/app success)
- [ ] P2 (Important: known docs, feature requests, or known bugs)
- [ ] P3 (Nice-to-have: docs or features that improve a flow)

#### Also, feel free to comment with your questions, concerns, thoughts, to volunteer yourself as a resource, or to tag in other people who may have more context!

**Next Steps**
- [x] Run Clay.Run to get company, title, emails, location, github, LinkedIns, and Twitters. Discord would be nice but it’s not ready. 

- [ ] Populate social channels from  “Is there anything else that you would like to share with us? Social media channels, some of your work, information that may be important, etc.?” to the appropriate fields (ie: telegram + Twitter, Github, etc)

- [x] Properly refactor answers from  “How do you see yourself contributing to NEAR?” to specific tags.

- [ ] Assign guild applicants to guilds

- [ ] Deduplicate 885 accounts  

- [ ] Find way to autoport and tag GitHub contributors depending on project submission (ie: “Bounties”, “Protocol”, “Tooling”, “Application” depending on which repos they engage with. 

- [ ] Find way to autoport in “interactions” from Github and Events 

- [ ] Add Airtable to Hopin + Crowdcast integration 

- [ ] Move to next step in Individual Contributor Strategy

**Comment by crasskitty** [comment]
Agree, this is a P1, X-Large T-shirt Epic.

**Comment by crasskitty** [comment]
@cjd425 and I completed these two steps:

 - Properly refactor answers from “How do you see yourself contributing to NEAR?” to specific tags.

**Comment by cameron-NEAR** [comment]
I'm blocked by legal right now because we can't contact anyone who we haven't gotten explicit consent from. More info TBA
